L1Trigger/L1THGCal/interface/concentrator/HGCalConcentratorBestChoiceImpl.h

```diff
@@ -19,10 +19,10 @@ class HGCalConcentratorBestChoiceImpl {
 
 private:
   std::vector<unsigned> nData_;
-  static constexpr unsigned kNDataSize_ = 64;
-  static constexpr uint32_t kWaferOffset_ = 3;
-  static constexpr uint32_t kWaferMask_ = 0x7;
-  static constexpr uint32_t kLinkMask_ = 0x7;
+  static const unsigned kNDataSize_ = 128;
```

Comment:
@jbsauvan @rekovic why was constexpr removed?

Reply:
Thanks for noticing. Looks like I had modified these lines when `const` was used and when I rebased I missed the `constexpr`. I will change back to `constexpr`

Reply:
@jbsauvan please propose it in a small update PR, I will merge this one for the time being

Reply:
ok, will do